ext/openssl/openssl.stub.php

```diff
@@ -177,15 +177,15 @@ function openssl_verify(string $data, string $signature, $public_key, string|int
 /**
  * @param string $sealed_data
  * @param array $encrypted_keys
- * @param string $initialization_vector
+ * @param string $iv
```

Comment:
what's the actual point of adding params to docblock if some functions don't have all params in docblocks (e.g. `openssl_open`)

Reply:
Docblock types are only added when the parameter type can't be added as a type declaration. It's always the case for output, by-ref parameters (like the ones here), but also the case for normal parameters where the type declaration is not in line with parameter parsing in the C code (because debug builds verify this).

Reply:
ah ok